modules/terraform/cmd.go

```diff
@@ -49,8 +52,16 @@ func GetCommonOptions(options *Options, args ...string) (*Options, []string) {
 		options.TerraformBinary = DefaultExecutable
 	}
 
-	if options.TerraformBinary == "terragrunt" {
+	if options.TerraformBinary == TerragruntDefaultPath {
 		args = append(args, "--terragrunt-non-interactive")
+		// for newer Terragrunt version, setting simplified log formatting
+		if options.EnvVars == nil {
+			options.EnvVars = map[string]string{}
+		}
+		_, tgLogSet := options.EnvVars["TERRAGRUNT_DISABLE_LOG_FORMATTING"]
+		if !tgLogSet {
+			options.EnvVars["TERRAGRUNT_DISABLE_LOG_FORMATTING"] = "true"
```

Comment:
This env var is deprecated, please use `TERRAGRUNT_LOG_FORMAT=key-value`

*But only if you use the latest version that supports log format.